modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -497,6 +497,7 @@ export interface WalletData {
   };
   multisigType: 'onchain' | 'tss';
   type?: WalletType;
+  tokens: Record<string, any>[];
```

Comment:
Will a tokens array always be returned all all coin wallets?

Reply:
craaap good catch, forgot to make optional